WultraMobileTokenSDK/Operations/Model/UserOperation/WMTTemplates.swift

```diff
@@ -0,0 +1,222 @@
+//
+// Copyright 2024 Wultra s.r.o.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions
+// and limitations under the License.
+//
+
+import Foundation
+
+/// Detailed information about displaying operation data
+///
+/// Contains prearranged styles for the operation attributes for the app to display
+public class WMTTemplates: Codable {
+    
+    /// The template how the operation should look like in the list of operations
+    public let list: ListTemplate?
+    
+    /// The template for how the operation data should look like
+    public let detail: DetailTemplate?
+    
+    // MARK: - Internals
+    
+    private enum Keys: String, CodingKey {
+        case list, detail
+    }
+    
+    public required init(from decoder: Decoder) throws {
+        let c = try decoder.container(keyedBy: Keys.self)
+        
+        list = try? c.decode(ListTemplate.self, forKey: .list)
+        detail = try? c.decode(DetailTemplate.self, forKey: .detail)
+    }
+    
+    public init(list: ListTemplate?, detail: DetailTemplate?) {
+        self.list = list
+        self.detail = detail
+    }
+
+    /// This typealias specifies that attributes using it should refer to `WMTOperationAttributes`.
+    ///
+    /// AttributeId is supposed to be `WMTOperationAttribute.AttributeLabel.id`
```

Comment:
This is an incorrect description because the type itself does not specify it, but the value that typed like reference it.

We need to explain it properly to not confuse anybody using it.

Example (but it needs improvements)

> > Values of the `AttributeId` typealias should be referencing existing `WMTOperationAttribute` by `WMTOperationAttribute.AttributeLabel.id`